R/stan_betareg.fit.R

```diff
@@ -20,7 +20,7 @@
 
 stan_betareg.fit <- function (x, y, z = NULL, weights = rep(1, NROW(x)), offset = rep(0, NROW(x)),
                               link = c("logit", "probit", "cloglog", "cauchit", "log", "loglog"), 
-                              link.phi = c("log", "identity", "sqrt"), ...,
+                              link.phi = "log", ...,
```

Comment:
Sorry if my comments from earlier today were confusing. I was trying to say that I don't think basing anything on the length of `link.phi` is the right way to go because that means the argument with  multiple options is treated differently than is typically expected.